WordPress/Classes/ViewRelated/Menus/MenuHeaderViewController.m

```diff
@@ -139,4 +142,36 @@ - (void)selectionViewSelectedOptionForCreatingNewItem:(MenusSelectionView *)sele
     [self contractSelectionsIfNeeded];
 }
 
+#pragma mark - accessibility
+
+- (void)prepareForVoiceOver
+{
+    [self configureLocationAccessibility];
+    [self configureMenuAccessibility];
+}
+
+- (void)configureLocationAccessibility
+{
+    // Menu area: Header. 3 menu areas available. Button. [hint] Expands to select a different menu area.
+    NSString *format = NSLocalizedString(@"Menu area: %@, %d menu areas available", @"Screen reader string too choose a menu area to edit. %@ is the name of the menu area (Primary, Footer, etc...). %d is a number indicating the ammount of menu areas available.");
+    MenusSelectionItem *selectedItem = self.locationsView.selectedItem;
+    self.locationsView.accessibilityLabel = [NSString stringWithFormat:format,
+                                                        selectedItem.displayName,
+                                                        self.blog.menuLocations.count];//self.locationsView.subTitleLabel.text];
+    self.locationsView.accessibilityHint = NSLocalizedString(@"Expands to select a different menu area", @"Screen reader hint (non-imperative) about what does the site menu area selector button do.");
```

Comment:
From what I see here the accessibility hint is set irrespective of whether the item is selected or not, and I'm seeing some slightly inconsistent behavior. 
Here's the sequence of steps I do (with VoiceOver on):
1. Single tap the menu location button (`MenusSelectionDetailView`) to hear: the accessibility label plus the "Expands to select a different menu area" accessibility hint ✅
2. Double tap the same button to expand the options, now I hear: the accessibility label plus the accessibility value "Expanded" ✅
3. Single tap the same button (just to hear VoiceOver read it again), and I hear a different message: the accessibility label plus the "Expands to select a different menu area" accessibility hint ⚠️

That final message doesn't sound right, it should read the same message as in Step 2. Here's a video that includes sound (VoiceOver audio included): https://cloudup.com/cVFQgi9dJGV (31s)

Reply:
Thank you for noticing this!

For the hint to be the same is fine as long as the status shows that the control is already expanded. In this case, when I test the point 3, VoiceOver does say the value `Expanded` so I'd say it's Ok.

After performing the action, is normal for VoiceOver to skip reading the Hint.

Reply:
Ok thanks for clarifying 👍